app/src/pages/playground/PlaygroundOutput.tsx

```diff
@@ -305,6 +310,11 @@ function PlaygroundOutputText(props: PlaygroundInstanceProps) {
     onCompleted: () => {
       markPlaygroundInstanceComplete(props.playgroundInstanceId);
     },
+    onFailed: () => {
+      // TODO(apowell): display error message?
+      // TODO(apowell): clear run id?
```

Comment:
I think yes, clear the runId

you can use this hook useNotifyError to just show an error toast for now.

Reply:
Done